source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+Once that is complete you can proceed with deployment of your MPNS instance.
+
+The documentation topics listed below guide installation and configuration for your MPNS.
+
+- `Push Notification Service Installation <https://developers.mattermost.com/contribute/mobile/push-notifications/service/>`_
+- `Admin Configuration for Push Notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_
+- `Additional FAQs <https://docs.mattermost.com/mobile/mobile-faq.html#>`_
+
+************************************
+Updating Your Custom Mattermost Apps
+************************************
+
+While not part of your initial mobile deployment, you should consider a strategy for updating when new versions of the Mattermost Mobile applications are available. We highly recommend you update your custom apps for any security or service releases. At the same time, if you have updated the apps, prior to distribution, check any compatibility requirements for the mobile apps and the Mattermost server. 
+
+    **Note:** Only updating the mobile apps, or updating the mobile apps before the Mattermost server, can result in incompatibility.
+
+----
+
+####################################
+Appendix A: Troubleshooting and FAQs
+####################################
+
+We highly recommend you check out our `Mobile FAQ <https://docs.mattermost.com/mobile/mobile-faq.html>`_ and `Mobile Troubleshooting documentation <https://docs.mattermost.com/mobile/mobile-troubleshoot.html>`_. The most common questions we've received have been answered there. However, it’s important to call out a few common items customers run into.
+
+*******************************
+Data Security on Mobile Devices
+*******************************
+
+- `How is data handled on a device after an account is deleted? <https://docs.mattermost.com/mobile/mobile-faq.html#how-is-data-handled-on-mobile-devices-after-a-user-account-is-deactivated>`_
+- `What post metadata is sent in mobile push notifications? <https://docs.mattermost.com/mobile/mobile-faq.html#what-post-metadata-is-sent-in-mobile-push-notifications>`_
+- `What are my options for securing the mobile apps? <https://docs.mattermost.com/mobile/mobile-faq.html#what-are-my-options-for-securing-the-mobile-apps>`_
+- `What are my options for securing push notifications? <https://docs.mattermost.com/mobile/mobile-faq.html#what-are-my-options-for-securing-push-notifications>`_
+
+***********************
+Corporate Proxy Servers
+***********************
+
+`How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server? <https://docs.mattermost.com/mobile/mobile-faq.html#how-do-i-receive-mobile-push-notification-if-my-it-policy-requires-the-use-of-a-corporate-proxy-server>`_
+
+- `Deploy Mattermost in a proxy-aware configuration with a pre-proxy relay <https://docs.mattermost.com/mobile/mobile-faq.html#deploy-mattermost-in-a-proxy-aware-configuration-with-a-pre-proxy-relay>`_
```

Comment:
I'm working with Sebastian to try and remove this from docs, so could we not link to it here if possible?

Reply:
+1